src/main/java/org/opentripplanner/model/TimetableSnapshot.java

```diff
@@ -604,6 +600,12 @@ private void swapTimetable(TripPattern pattern, Timetable original, Timetable up
     dirty = true;
   }
 
+  private void validateReadOnly() {
```

Comment:
I would call this `validateNotReadOnly()` or `validateWritable()` or `throwIfReadOnly()`

Reply:
Done